cypress/integration/with-users/legacy/auth.spec.ts

```diff
@@ -1,14 +0,0 @@
-import { generateMAASURL } from "../../utils";
-
-context("Legacy authentication", () => {
```

Comment:
The legacy client no longer needs to redirect to the React client to log in. Login tests are covered in login.spec.js.